### ISSUE TITLE
Cinnamon always crash when I start the computer

### ISSUE THREAD

**Issue by jorgesumle** [linuxmint/Cinnamon#5720]
I have been using Linux Mint 18 Sarah for two or three months. Until now, I haven't had any issue. But recently I got the message Cinnamon just crashed. You are currently running in Fallback mode. If I say 'Yes' to restart Cinnamon the mouse centers and I get the same error message. I wasn't doing nothing strange when the error happened.

I'm just wondering how can I restart Cinnamon correctly or reset the configuration to the default to see if I can fix the problem. I can get to the tty terminals, so I can type commands. Please tell me how can I display the error log, so that I can include it in this issue and you can determine the problem.

![20160923_141955](https://cloud.githubusercontent.com/assets/14212780/18808999/76afa6de-8271-11e6-9b1d-c9bfb5c50a62.jpg)

**Comment by mtwebster** [comment]
Can you try to log in with software rendering?  In the login screen, select a different session - there will be Cinnamon and Cinnamon (Software Rendering) or something similar.  This will help eliminate if this is some graphics issue or not..

Have you recently changed any drivers, or installed a new applet?  You can check .xsession-errors in your home folder for clues as to what might be happening.

To reset all of cinnamon's settings (last resort) - go into a tty (ctr-alt-f1 or -f2 etc...) login and run:

`gsettings reset-recursively org.cinnamon`

**Comment by jorgesumle** [comment]
> Can you try to log in with software rendering? In the login screen, select a different session - there will be Cinnamon and Cinnamon (Software Rendering) or something similar. This will help eliminate if this is some graphics issue or not..

I cannot select a session, because Cinnamon crashes before loading the login screen. Is there a way I can test that from any tty terminal?

> 
Have you recently changed any drivers, or installed a new applet? You can check .xsession-errors in your home folder for clues as to what might be happening.

No. I'll check .xsession-errors on Sunday. Now I have no access to that computer.

> To reset all of cinnamon's settings (last resort) - go into a tty (ctr-alt-f1 or -f2 etc...) login and run:
> 
> gsettings reset-recursively org.cinnamon

OK, I'll do that after figuring out and telling you what failed.

**Comment by jorgesumle** [comment]
I managed to copy the file .xsession-errors to an USB. Here is the output:




initctl: Unable to connect to Upstart: Failed to connect to socket /com/ubuntu/upstart: Connection refused
/etc/mdm/Xsession: Beginning session setup...
localuser:jorge being added to access control list
** Message: couldn't access control socket: /run/user/1000/keyring/control: No such file or directory
** Message: couldn't access control socket: /run/user/1000/keyring/control: No such file or directory
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh

(cinnamon-settings-daemon:1943): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.
=== xinerama setup Configuration ===
  Clone: false
  Output: (null) attached to HDMI-0
     status: off
     width: -1
     height: -1
     rate: -1
     primary: true
     position: -1 -1
  Output: Laptop attached to eDP
     status: on
     width: 1366
     height: 768
     rate: 60
     primary: true
     position: 0 0
=== Applying Configuration Configuration ===
  Clone: false
  Output: (null) attached to HDMI-0
     status: off
     width: -1
     height: -1
     rate: -1
     primary: false
     position: -1 -1
  Output: Laptop attached to eDP
     status: on
     width: 1366
     height: 768
     rate: 60
     primary: true
     position: 0 0
cinnamon-session[1837]: GdkPixbuf-WARNING: t+4,43994s: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(cinnamon:2118): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(cinnamon:2118): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_height: assertion 'GDK_IS_PIXBUF (pixbuf)' failed

(cinnamon:2118): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_width: assertion 'GDK_IS_PIXBUF (pixbuf)' failed
Window manager warning: Log level 8: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(cinnamon:2118): GdkPixbuf-CRITICAL **: gdk_pixbuf_loader_write: assertion 'error == NULL || *error == NULL' failed
Window manager warning: Failed to load theme "Mint-Y-Dark": Line 199 character 172: Couldn't recognize the image file format for file '/usr/share/themes/Mint-Y-Dark/metacity-1/button-bg.svg'
Window manager warning: Could not find theme Mint-Y-Dark. Falling back to default theme./usr/lib/blueberry/blueberry-tray.py:5: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Gdk, GnomeBluetooth, Gio
/usr/lib/blueberry/blueberry-tray.py:5: PyGIWarning: GnomeBluetooth was imported without specifying a version first. Use gi.require_version('GnomeBluetooth', '1.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Gdk, GnomeBluetooth, Gio

(blueberry-tray.py:2134): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(polkit-gnome-authentication-agent-1:2138): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(nemo:2131): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.
Cjs-Message: JS LOG: About to start Cinnamon

(nm-applet:2133): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(nm-applet:2133): nm-applet-WARNING **: failed loading default-icon: Image type 'png' is not supported
sys:1: PyGIWarning: Nemo was imported without specifying a version first. Use gi.require_version('Nemo', '3.0') before import to ensure that the right version gets loaded.

(nm-applet:2133): nm-applet-WARNING **: failed to load icon "nm-signal-100": Unrecognized image file format
/usr/lib/python2.7/dist-packages/glipper/AppIndicator.py:41: GtkWarning: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.
  self._app_indicator = appindicator.Indicator("glipper", "glipper", appindicator.CATEGORY_APPLICATION_STATUS)

(nm-applet:2133): nm-applet-WARNING **: failed to load icon "nm-device-wired": Unrecognized image file format

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): nm-applet-WARNING **: failed to load icon "nm-signal-50": Unrecognized image file format

(nm-applet:2133): nm-applet-WARNING **: failed to load icon "nm-signal-75": Unrecognized image file format

(nm-applet:2133): nm-applet-WARNING **: failed to load icon "nm-adhoc": Unrecognized image file format
/usr/bin/glipper:45: GtkWarning: Error loading theme icon 'image-missing' for stock: Unrecognized image file format
  gtk.main()
St-Message: cogl npot texture sizes SUPPORTED
Cjs-Message: JS LOG: Cinnamon started at Mon Sep 26 2016 00:12:25 GMT+0200 (CEST)

(cinnamon:2118): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed
Cjs-Message: JS LOG: network applet: Cannot find connection for active (or connection cannot be read)
Nemo-Share-Message: Called "net usershare info" but it failed: 'net usershare' returned error 255: mkdir failed on directory /var/run/samba/msg.lock: Permission denied
net usershare: cannot open usershare directory /var/lib/samba/usershares. Error No such file or directory
Please ask your system administrator to enable user sharing.


(cinnamon:2118): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2118): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed
openGL version 4.1 detected (GL3 Cogl Driver)
MetaSyncRing disabled: couldn't find required GL extensions, or the minimum safe openGL version was not met
**
Gtk:ERROR:/build/gtk+3.0-6ZPWga/gtk+3.0-3.18.9/./gtk/gtkicontheme.c:4203:gtk_icon_info_load_icon_finish: assertion failed: (icon_info_get_pixbuf_ready (icon_info))

(cinnamon-launcher:2109): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format
/usr/bin/cinnamon-launcher:18: Warning: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
  d.show_all()

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format
/usr/bin/cinnamon-launcher:19: Warning: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
  resp = d.run()

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2109): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(metacity:2196): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(gnome-panel:2198): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.
Window manager warning: Failed to load theme "Mint-X": Line 251 character 26: Couldn't recognize the image file format for file '/usr/share/themes/Mint-X/metacity-1/menu-focus-normal.svg'

** (gnome-panel:2198): WARNING **: Failed to register client: GDBus.Error:org.gnome.SessionManager.AlreadyRegistered: Unable to register client
**
metacity:ERROR:ui/ui.c:624:meta_ui_get_default_window_icon: assertion failed: (default_icon)
cinnamon-session[1837]: WARNING: t+11,37065s: App 'cinnamon.desktop' respawning too quickly
cinnamon-session[1837]: CRITICAL: t+11,37160s: We failed, but the fail whale is dead. Sorry....
cinnamon-session[1837]: WARNING: t+11,37254s: Application 'cinnamon.desktop' killed by signal 15
cinnamon-session[1837]: WARNING: t+11,37299s: App 'cinnamon.desktop' respawning too quickly

(cinnamon:2210): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(cinnamon:2210): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_height: assertion 'GDK_IS_PIXBUF (pixbuf)' failed

(cinnamon:2210): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_width: assertion 'GDK_IS_PIXBUF (pixbuf)' failed
Window manager warning: Log level 8: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(cinnamon:2210): GdkPixbuf-CRITICAL **: gdk_pixbuf_loader_write: assertion 'error == NULL || *error == NULL' failed
Window manager warning: Failed to load theme "Mint-Y-Dark": Line 199 character 172: Couldn't recognize the image file format for file '/usr/share/themes/Mint-Y-Dark/metacity-1/button-bg.svg'
Window manager warning: Could not find theme Mint-Y-Dark. Falling back to default theme.Cjs-Message: JS LOG: About to start Cinnamon
St-Message: cogl npot texture sizes SUPPORTED
Cjs-Message: JS LOG: Cinnamon started at Mon Sep 26 2016 00:12:29 GMT+0200 (CEST)

(cinnamon:2210): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed
Cjs-Message: JS LOG: network applet: Cannot find connection for active (or connection cannot be read)

(cinnamon:2210): GLib-GIO-CRITICAL **: g_file_new_for_path: assertion 'path != NULL' failed

(cinnamon:2210): GLib-GIO-CRITICAL **: g_loadable_icon_load: assertion 'G_IS_LOADABLE_ICON (icon)' failed
openGL version 4.1 detected (GL3 Cogl Driver)
MetaSyncRing disabled: couldn't find required GL extensions, or the minimum safe openGL version was not met
**
Gtk:ERROR:/build/gtk+3.0-6ZPWga/gtk+3.0-3.18.9/./gtk/gtkicontheme.c:4203:gtk_icon_info_load_icon_finish: assertion failed: (icon_info_get_pixbuf_ready (icon_info))

(cinnamon-launcher:2209): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(metacity:2226): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(gnome-panel:2230): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.
Window manager warning: Failed to load theme "Mint-X": Line 251 character 26: Couldn't recognize the image file format for file '/usr/share/themes/Mint-X/metacity-1/menu-focus-normal.svg'

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format
/usr/bin/cinnamon-launcher:18: Warning: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
  d.show_all()

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.

** (gnome-panel:2230): WARNING **: Failed to register client: GDBus.Error:org.gnome.SessionManager.AlreadyRegistered: Unable to register client
**
metacity:ERROR:ui/ui.c:624:meta_ui_get_default_window_icon: assertion failed: (default_icon)
cinnamon-session[1837]: WARNING: t+14,52740s: App 'cinnamon.desktop' respawning too quickly
cinnamon-session[1837]: WARNING: t+14,52783s: App 'cinnamon.desktop' respawning too quickly

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format
/usr/bin/cinnamon-launcher:19: Warning: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
  resp = d.run()

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-launcher:2209): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(cinnamon-screensaver:2249): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
mintUpdate: no process found

(mintUpdate.py:2257): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(mintUpdate.py:2257): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unrecognized image file format

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(cinnamon-settings-daemon:1943): color-plugin-WARNING **: failed to connect to device: Failed to connect to missing device /org/freedesktop/ColorManager/devices/cups_Brother_MFC_8880DN

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(nm-applet:2133): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

**Comment by jorgesumle** [comment]
gsettings reset-recursively org.cinnamon

gives me

(process:4306): dconf-WARNING **: failed to commit changes to dconf: Cannot autolaunch D-Bus without X11 $DISPLAY

It hasn't fixed anything. I restarted the computer and the same problem occurs.

**Comment by jorgesumle** [comment]
The problem is fixed now that I entered recovery mode and selected the option to check or repair pkgs (or something similar, I don't remember the name). I'm going to close the issue because my problem was solved and I'm not sure if the problem is caused by Linux Mint or by Cinnamon. You have the output of the error in my recent post, you probably understand it better than me, so if this is an issue of Cinnamon you should re-open the issue. Thanks!

**Comment by craftyguy** [comment]
@jorgesumle 
How did you resolve this? What "recovery mode" are you referring to?

**Comment by jorgesumle** [comment]
I pressed the Escape key to open the start-up menu, there are three options in my case:
1. Linux Mint 18 Cinnamon 64-bit
2. Advanced options for Linux Mint 18 Cinnamon 64-bit
3. System setup

In the advanced options there mode is a recovery mode option. I clicked that and entered a _recovery menu_ with various options:
1. resume: Resume normal boot
2. clean: Try to make free space
**3. dpkg: Repair broken packages**
4. fsck: Check all file systems
5. grub ...
...

Just chose the 3rd option from the _recovery menu_ and it fixed, but I still don't know what was the problem. All I know is that the computer started to work again and I've haven't experienced any problem since then.
